src/dotnet-install.ps1

```diff
@@ -859,6 +870,9 @@ function Get-AkaMSDownloadLink([string]$Channel, [string]$Quality, [string]$Prod
     {
         try {
             $akaMsDownloadLink = $Response.Headers.GetValues("Location")[0]
+            #FeedCredential is the part of redirect link, remove it
+            $akaMsDownloadLink = $akaMsDownloadLink.split('?')[0]
```

Comment:
I removed the credentials from the redirect link here to avoid double application of credentials while the actual downloading. However, might be better to not apply credentials to the aka.ms link in the first place, i.e. have a bool parameter with the meaning if it is needed to apply credentials or not in GetHTTPResponse function.